### PR TITLE
Removes stealth attack from hunter

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Removes hunter stealth attack

## Why It's Good For The Game

This shit is a meme,everyone hates it,begone

## Changelog
:cl:
balance: Remove hunter stealth attack
/:cl:

Closes #3439 
Closes #3440

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by adamkad1** [comment]
okay, remove all marine stealths then

**Comment by Kuro-ai** [comment]
> okay, remove all marine stealths then

Last I checked marines dont gain additional damage,knockout and armor pen when stealthed and also xenos have xray

**Comment by adamkad1** [comment]
> > okay, remove all marine stealths then
> 
> Last I checked marines dont gain additional damage,knockout and armor pen when stealthed and also xenos have xray

xray doesnt make stelfed marines not be stelfy

**Comment by Kuro-ai** [comment]
Xray makes marines really easy to see,the hunter on the other hand isnt,and stealth attack is not good for the game so its getting removed

**Comment by spookydonut** [comment]
Removing stealth attack without giving them something else to replace it is short sighted.

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 1]
```diff
@@ -69,18 +68,4 @@
...
-		X.can_sneak_attack = FALSE
-		addtimer(CALLBACK(X, /mob/living/carbon/xenomorph/hunter/.proc/sneak_attack_cooldown), HUNTER_POUNCE_SNEAKATTACK_DELAY)
-
-/mob/living/carbon/xenomorph/hunter/proc/sneak_attack_cooldown()
-	if(!stealth || can_sneak_attack)
-		return
-	can_sneak_attack = TRUE
-	to_chat(src, "<span class='xenodanger'>We're ready to use Sneak Attack while stealthed.</span>")
-	playsound(src, "sound/effects/xeno_newlarva.ogg", 25, 0, 1)
+		owner.update_icons()
```
Comment (spookydonut) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm]:
```suggestion
		owner.update_icons()

```

**Comment by Rohesie** [comment]
This does not fully remove it, it leaves a lot of stealth attack stuff in the code.

Stealth attack is a big balance concern at the moment, with increased walk speed. It needs to be substituted by a better concept.

**Comment by GravitronBlue** [comment]
Start with nerfing the hunter's ability to use stealth while at running speed, and consider taking away the stun from it, a longer cooldown on stealth strikes would be nice.

**Comment by spookydonut** [comment]
#3390

**Comment by spookydonut** [comment]
#3439

**Comment by spookydonut** [comment]
#3439 
#3440

**Pull request closed without merging**